packages/editor-ui/src/components/SqlEditor/SqlEditor.vue

```diff
@@ -20,28 +28,61 @@ import {
 	keymap,
 	lineNumbers,
 } from '@codemirror/view';
-import { MSSQL, MySQL, PostgreSQL, sql, StandardSQL } from '@codemirror/lang-sql';
-import type { SQLDialect } from 'n8n-workflow';
-
+import {
+	keywordCompletion,
+	MSSQL,
+	MySQL,
+	PostgreSQL,
+	schemaCompletion,
+	StandardSQL,
+	MariaSQL,
+	SQLite,
+	Cassandra,
+	PLSQL,
+} from './sql-parser-skipping-whitespace';
+import type { SQLDialect as SQLDialectType } from './sql-parser-skipping-whitespace';
 import { codeNodeEditorTheme } from '../CodeNodeEditor/theme';
+import { n8nCompletionSources } from '@/plugins/codemirror/completions/addCompletions';
+import { expressionInputHandler } from '@/plugins/codemirror/inputHandlers/expression.inputHandler';
+import { highlighter } from '@/plugins/codemirror/resolvableHighlighter';
+import { expressionManager } from '@/mixins/expressionManager';
+import InlineExpressionEditorOutput from '@/components/InlineExpressionEditor/InlineExpressionEditorOutput.vue';
+import { EXPRESSIONS_DOCS_URL } from '@/constants';
+import type { TargetItem } from '@/Interface';
 
 const SQL_DIALECTS = {
-	standard: StandardSQL,
-	mssql: MSSQL,
-	mysql: MySQL,
-	postgres: PostgreSQL,
+	StandardSQL,
+	PostgreSQL,
+	MySQL,
+	MariaSQL,
+	MSSQL,
+	SQLite,
+	Cassandra,
+	PLSQL,
```

Comment:
Do we support `MariaSQL`, `SQLite`, `Cassandra`, `PLSQL`?

Reply:
Not right now. I added them since codemirror supports it.